bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/Zh_M125_ToAA_Tobbgg_LO_MLM/Zh_M125_ToAA__AMASS__Tobbgg_proc_card.dat

```diff
@@ -0,0 +1,6 @@
+import model --modelname HAHM_MG5model_v3/HAHM_variableMW_v3_UFO
+define hs1 = hs
+define hs2 = hs
```

Comment:
what is the reason for these definitions ?

Reply:
You mean the gridpack generation fails if the hs are not defined twice like above?

Reply:
Yes, correct, it can not catch the processes correctly.. So, it's done..